project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.17
```

Comment:
This line is breaking everything for me. As far as I can see version 0.13.18 doesn't exist? https://www.scala-sbt.org/download.html